### DISCUSSION TITLE
v3.5 community license

### DISCUSSION THREAD

**Discussion by edbnewbie** [ravendb/ravendb#19013] [General]
Hi, When I sign up for a community license, I get the license for v6. How do I get the community license for v3.5. If thats not an option, whats the latest version of Ravendb I need to be on to be eligible for a community license intended for production use

**Comment by ayende** [comment]
Version 3.5 doesn't have a community license, that is available from version 4.0 onward.

Note that version 3.5 is now also no longer supported at all.

What is your scenario for needing it?

**Reply by edbnewbie** [reply]
We are currently unable to upgrade to v4 due to the extensive code changes needed for the upgrade.  I will talk to the team internally. If community license is available for production use for v4 , how do we get it for a specific version ?

**Reply by ayende** [reply]
Community license is available from v4.0 _and upward_.
For version 3.5 - you need to contact us, see: https://ravendb.net/contact
